fxapom/fxapom.py

```diff
@@ -41,7 +47,6 @@ class FxATestAccount:
 
     def __init__(self, url=DEV_URL):
         """ Creates an FxATestAccount object, which includes a verified account.
-
```

Comment:
Please restore this blank line. See http://legacy.python.org/dev/peps/pep-0257/#multi-line-docstrings for details.